src/sentry/tasks/post_process.py

```diff
@@ -216,10 +218,15 @@ def process_snoozes(group):
     """
     from sentry.models import GroupSnooze, GroupStatus
 
-    try:
-        snooze = GroupSnooze.objects.get_from_cache(group=group)
-    except GroupSnooze.DoesNotExist:
-        return False
+    key = "postprocess_gs:1:%s" % (group.id)
+    snooze = cache.get(key)
+    if not snooze:
+        try:
+            snooze = GroupSnooze.objects.get(group=group)
+            cache.set(key, snooze, 3600)
+        except GroupSnooze.DoesNotExist:
+            cache.set(key, 0, 60)
```

Comment:
Not sure how I feel about this... This means that someone can snooze a group and have it immediately alert them again. Will be especially likely on high volume groups.

Is this reasonable case for active cache invalidation?

Reply:
The entire reason for caching this is to stop us from doing a direct lookup when a GroupSnooze does not exist. There is a 0.01% chance that we will actually get a hit from the `get_from_cache` lookup. Is there a better way to do a negative lookup here?

Reply:
We could cache this, and then when someone creates a `GroupSnooze` we delete this cache value. You could then set this for a longer period of time as well.

Active cache can be tricky though, but maybe we can just have it on a `post_save` signal for `GroupSnooze`